lib/manageiq_performance/configuration.rb

```diff
@@ -143,4 +143,16 @@ def browser_mode_config(opts={})
   def self.config
     @config ||= Configuration.load_config
   end
+
+  def self.config= config
+    @config = Configuration.new config
+  end
+
+  def self.with_temporary_config temporary_config
```

Comment:
I like that.  I can just explain in a comment that this is temporary.  Good call 👍